### PR TITLE
Discarding staged changes vs unstaged changes - commands corrected

### PR DESCRIPTION
**Pull request by flexdinesh**
Few commands fixed in Staging section - discarding Staged changes vs Unstaged changes.
Reference: https://stackoverflow.com/a/22621464

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,17 +293,12 @@ $ git stash pop
 ```
 
 <a href="i-want-to-discard-my-local-uncommitted-changes"></a>
-### I want to discard my local, uncommitted changes
+### I want to discard my local uncommitted changes(staged and unstaged)
```
Comment (RichardLitt) [inline README.md]: Add a space between changes and the parens

File: README.md [hunk 2]
```diff
@@ -357,14 +352,23 @@ Alternatively, to discard multiple files in your working copy, list them all.
...
+### I want to discard only my unstaged local changes
 
-When you want to get rid of all of your local uncommitted changes
+When you want to get rid of all of your unstaged local uncommitted changes
 
 ```sh
 $ git checkout .
 ```
 
+### I want to discard all of my untracked files
```
Comment (RichardLitt) [inline README.md]: Needs the `<a>` tags

Comment (RichardLitt) [reply README.md]: Also needs to be added to the TOC

File: README.md [hunk 3]
```diff
@@ -357,14 +352,23 @@ Alternatively, to discard multiple files in your working copy, list them all.
...
+
+When you want to get rid of all of your untracked files
+
+```sh
+$ git clean -f
```
Comment (RichardLitt) [inline README.md]: Why not `-d` and `-x`, too?

Comment (radiumsharma) [reply README.md]: I have made the other two changes. I will send out a new commit for -x and -d flags during checkout

Comment (flexdinesh) [reply README.md]: I have made the other two changes. I will send out a new commit for -x and -d flags during checkout

File: README.md (hunk 1 above)
Comment (flexdinesh) [reply README.md]: This is done

File: README.md (hunk 2 above)
Comment (flexdinesh) [reply README.md]: This is done.

File: README.md (hunk 3 above)
Comment (RichardLitt) [reply README.md]: What do you mean by checkout?

Comment (flexdinesh) [reply README.md]: (checkout .) gets the latest files from remote and disregards local unstaged changes. This is a common practice to discard local changes to restore working directory to original state.

Comment (flexdinesh) [reply README.md]: If you're asking about checkout -f, then no, I did not add that and just left it there as it was already there.

File: README.md [hunk 4]
```diff
@@ -36,10 +36,11 @@ For clarity's sake all examples in this document use a customized bash prompt in
   - [Unstaged Edits](#unstaged-edits)
     - [I want to move my unstaged edits to a new branch](#i-want-to-move-my-unstaged-edits-to-a-new-branch)
     - [I want to move my unstaged edits to a different, existing branch](#i-want-to-move-my-unstaged-edits-to-a-different-existing-branch)
-    - [I want to discard my local, uncommitted changes](#i-want-to-discard-my-local-uncommitted-changes)
+    - [I want to discard my local uncommitted changes (staged and unstaged)](#i-want-to-discard-my-local-uncommitted-changes)
```
Comment (RichardLitt) [inline README.md]: Will this link still work?

File: README.md [hunk 5]
```diff
@@ -293,7 +294,14 @@ $ git stash pop
...
 <a href="i-want-to-discard-my-local-uncommitted-changes"></a>
-### I want to discard my local, uncommitted changes
+### I want to discard my local uncommitted changes (staged and unstaged)
+
+If you want to discard all your local staged and unstaged changes, you can do this:
+
+```sh
+(my-branch)$ git reset --hard
+# or
+(master)$ git checkout -f
```
Comment (RichardLitt) [inline README.md]: Closing ticks?

File: README.md (hunk 4 above)
Comment (flexdinesh) [reply README.md]: The link should work. We have a reference to that in the explanation section.

File: README.md (hunk 5 above)
Comment (flexdinesh) [reply README.md]: Added

**Comment by RichardLitt** [comment]
Looks good to me! Thank you!

**Pull request merged**